financial-connections-example/build.gradle

```diff
@@ -9,6 +9,9 @@ android {
 
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
 
+        versionName VERSION_NAME
+        versionCode getVersionCode(versionName)
```

Comment:
Just curious are you only releasing a build when a new release is cut? For our case, we sometimes need to manually release multiple ones per week during bugbash, adding this might limit the flexibility of releasing

Reply:
Connections does not currently release, and instead sends builds using Firebase. Firebase allows builds with the same version code / name to be distributed.